packages/ui-components/src/components/calendar/test/calendar.spec.tsx

```diff
@@ -1,19 +1,16 @@
 import { h, SpecPage } from '@stencil/core/internal';
 import { newSpecPage } from '@stencil/core/testing';
 import { KvCalendar } from '../calendar';
-import { getDateMonth, getDateYear } from '../../../utils/date.helper';
 
 describe('Calendar (unit tests)', () => {
 	let page: SpecPage;
 	let component: KvCalendar;
 
 	describe('when uses default props', () => {
-		const now = new Date();
-
 		beforeEach(async () => {
 			page = await newSpecPage({
 				components: [KvCalendar],
-				html: '<kv-calendar></kv-calendar>'
+				template: () => <kv-calendar initialDate="2022-08-01"></kv-calendar>
```

Comment:
You can use JSX here like you did with `<kv-calendar initialDate="2022-04-01"></kv-calendar>`

I think there's no need to use JSX though, you could use the html prop `initial-date` (as stencil automatically converts prop names) because it's a string and not a complex object.

Reply:
I just saw the other usages, keep the consistency of JSX usage :)